api/Crypto.json

```diff
@@ -26,10 +26,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": "26"
+            "version_added": "1"
```

Comment:
This makes the data look weird. Was there nothing on the `window.crypto` object between Firefox 1 and 26? Or only things that aren't in BCD? Maybe just match the version of the earliest subfeature?